mne/channels/montage.py

```diff
@@ -302,6 +303,22 @@ def _get_dig_names(self):
 
         return dig_names
 
+    def compute_native_head_t(self):
+        """Compute the transformation to the head coordinate frame.
+
+        Returns
+        -------
+        tr : instance of Transform
+            The transformation. In the special case where the ``coord_frame``
+            during montage init was ``'mri'``, this will be a :term:`trans`
+            from the MRI to head coordinate frame.
+
+        Notes
+        -----
+        .. versionadded:: 0.21
+        """
+        return compute_native_head_t(self)
```

Comment:
shall we deprecate function then? so we have one way of doing it?

Reply:
I'm fine with leaving both. But really I guess I should kill this method and use the function, and then add the function to `python_reference.rst`